subcommands/status/cmd.go

```diff
@@ -15,6 +15,8 @@ var api *client.Api
 
 var inactiveThreshold int
 
+const untagged = "(Untagged)"
```

Comment:
I cannot believe that the linter is going to _force us_ decide what is a const and what is a string literal.
Is there a way to globally disable this kind of warning?

That said, if @doanac, @mike-sul, and @StealthyCoder beat me to tell that _this is nice_, I'm fine to try doing it this way going forward.

Reply:
> I cannot believe that the linter is going to force us decide what is a const and what is a string literal.
Usually any string literal that we use more than once could be an const
However, we can just not add the goconst check so that the linter will not comply about it.

I am fine with this one 👍 
I will create a PR with just the lints that does not raise any issue
and then a pr for each case scenario and drop this one. 
So that will be easier the review.